analyzers/tests/SonarAnalyzer.UnitTest/Helpers/SymbolHelperTest.cs

```diff
@@ -398,5 +398,185 @@ public void GetClassification_Method(MethodKind methodKind, string expected)
 
             fakeSymbol.Object.GetClassification().Should().Be(expected);
         }
+
+        [DataTestMethod]
+        [DataRow("BaseClass`1", "VirtualMethod", "MyInheritedAttribute", "MyDerivedInheritedAttribute", "MyNotInheritedAttribute", "MyDerivedNotInheritedAttribute", "MyUnannotatedAttribute")]
```

Comment:
Let's put `"MyNotInheritedAttribute"` last, it makes it easier to compare with the next line